src/tune-hyperparameters/src/test/scala/VerifyTuneHyperparameters.scala

```diff
@@ -143,8 +141,8 @@ class VerifyTuneHyperparameters extends Benchmarks {
     new TuneHyperparameters()
       .setModels(models.toArray)
       .setEvaluationMetric(MetricConstants.AccuracySparkMetric)
-      .setNumFolds(4)
-      .setNumRuns(models.length * 3)
+      .setNumFolds(2)
+      .setNumRuns(2)
```

Comment:
2 runs seems very low, I would keep it as models.length * 3

Reply:
or maybe models.length * 2 at least

Reply:
this takes a super long time to run, we just need to test the basic functionality here